test-utils/src/main/java/com/revenuecat/purchases/utils/productStubs.kt

```diff
@@ -38,6 +38,32 @@ fun stubStoreProduct(
     override fun writeToParcel(dest: Parcel?, flags: Int) {}
 }
 
+@SuppressWarnings("EmptyFunctionBlock")
+fun stubINAPPStoreProduct(
+    productId: String
+): StoreProduct = object : StoreProduct {
+    override val productId: String
+        get() = productId
+    override val type: ProductType
+        get() = ProductType.INAPP
+    override val oneTimeProductPrice: Price?
+        get() = null
```

Comment:
Hmm should this have a value? (Not needed for current uses but I think it would be good to add it)

Reply:
Oh yes, good catch